p2p/sync.go

```diff
@@ -64,7 +64,7 @@ func (s *syncService) start(ctx context.Context) {
 		var nextHeight int
 		if curHeight, err := s.blockchain.Height(); err == nil {
 			nextHeight = int(curHeight) + 1
-		} else if !errors.Is(db.ErrKeyNotFound, err) {
```

Comment:
another one 👍 , can you please check later is there a linter that can catch these errors?

Reply:
Yes, it's a [`staticcheck`](https://staticcheck.dev/docs/checks#SA1032)